### PR TITLE
update repo license to BSD3 and add CC-BY-SA 4.0

### PR DESCRIPTION
**Pull request by elioqoshi**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,9 +1,25 @@
...
+
 THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+---
+
+## CC-BY-SA 4.0 for graphical assets:
+
+Copyright 2017 Open Observatory of Network Interference (OONI), The Tor Project
+
+<WORK> is licensed under a
```
Comment (hellais) [inline LICENSE]: You should replace `<WORK>` with `ooni-design`

File: LICENSE [hunk 2]
```diff
@@ -1,9 +1,25 @@
-Copyright 2017 Open Observatory of Network Interference
+## BSD3 for code:
```
Comment (hellais) [inline LICENSE]: I would suggest not using markdown formatting as this is not a markdown file.

Also the license string is `BSD-3-Clause`.

I would say:

"Code is licensed as BSD-3-Clause:"

File: LICENSE [hunk 3]
```diff
@@ -1,9 +1,25 @@
...
 
+3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
+
 THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+---
+
+## CC-BY-SA 4.0 for graphical assets:
```
Comment (hellais) [inline LICENSE]: Same as above, I would say:

"Graphical assets are licensed as CC-BY-SA 4.0 International:"

**Comment by elioqoshi** [comment]
@hellais did the changes. Green light?

**Comment by hellais** [comment]
LGTM

**Pull request merged**